integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -860,7 +860,12 @@ public void createDomainOnExistingDirectory() throws Exception {
               + "\"";
       LoggerHelper.getLocal().log(Level.INFO,
           "making sure the domain directory exists by running " + cmd);
-      ExecResult result = TestUtils.exec(cmd);
+      // Looking for servers directory as sometimes krun.sh exits with non-zero
```

Comment:
these changes are already in develop branch, not sure why they are appearing in your PR. You may want to sync to latest develop branch and resolve conflicts.

Reply:
Just synced with latest develop branch